tools/pipeline/Main.cpp

```diff
@@ -37,7 +37,7 @@ static cl::list<string>
   InputPipeline("P", desc("<Pipeline>"), cat(PipelineCategory));
 
 static cl::list<string> Targets(Positional,
-                                OneOrMore,
+                                ZeroOrMore,
```

Comment:
What happens if there's zero targets and no `--analyze-all`?

Reply:
it terminates with great success